Compilation issue on 0.2.5

On Debian 8 (jessie) I'm having a compilation issue, see message below, this error doesn't occur if I pin to the previous release 0.2.4.
For info, I use esqlite elixir/sqlitex.

===> Compiling pc
===> Compiling esqlite
===> Compiling /app/deps/esqlite/c_src/esqlite3_nif.c
===> /app/deps/esqlite/c_src/esqlite3_nif.c: In function âmake_rowâ:
/app/deps/esqlite/c_src/esqlite3_nif.c:537:5: error: âforâ loop initial declarations are only allowed in C99 or C11 mode
     for(int i = 0; i < size; i++)
     ^
/app/deps/esqlite/c_src/esqlite3_nif.c:537:5: note: use option -std=c99, -std=gnu99, -std=c11 or -std=gnu11 to compile your code
/app/deps/esqlite/c_src/esqlite3_nif.c: At top level:
/app/deps/esqlite/c_src/esqlite3_nif.c:107:1: warning: âmake_row_tupleâ defined but not used [-Wunused-function]
 make_row_tuple(ErlNifEnv *env, ERL_NIF_TERM value)
 ^

Docker images -> https://github.com/areski/excdr-pusher/tree/master/docker

Let me know if I can provide more information to help solving this issue.
Thanks!

--- a/rebar.config
+++ b/rebar.config
@@ -1,11 +1,14 @@
 {require_min_otp_vsn, "R15"}.
 {port_env, [
-    %% Default darwin ldflags causes loading of system sqlite. Removed -bundle flag. 
-    {"darwin", "DRV_LDFLAGS", 
+    %% Default darwin ldflags causes loading of system sqlite. Removed -bundle flag.
+    {"darwin", "DRV_LDFLAGS",
         "-flat_namespace -undefined suppress $ERL_LDFLAGS"},
 
     {"solaris", "CFLAGS",
         "-std=c99"},
+
+    {"linux", "CFLAGS",
+        "-std=c11"},
 
     {"CFLAGS", "$CFLAGS -DSQLITE_THREADSAFE=1 -DSQLITE_USE_URI -DSQLITE_ENABLE_FTS3 -DSQLITE_ENABLE_FTS3_PARENTHESIS"}
 ]}.

